frontend/src/metabase/public/components/EmbedModal/StaticEmbedSetupPane/StaticEmbedSetupPane.styled.tsx

```diff
@@ -54,14 +54,17 @@ export const DisplayOptionSection = ({
   titleId,
   children,
 }: {
-  title: string;
+  title: string | null;
```

Comment:
you could revert this change since you already have `QuestionDownloadSettings` and `DashboardDownloadSettings`.

Reply:
Good catch! Lemme revert that

Reply:
Fixed in https://github.com/metabase/metabase/pull/56211/commits/c16e07601992bb8b9a8cae7dd9ee071e8cba645f